### DISCUSSION TITLE
CSSBattle seems gradually turning into Unicode-art Battle. Do we need to address it?

### DISCUSSION THREAD

**Discussion by SelenIT** [cssbattle/forum#335] [Ideas]
The excellent [table of the CSSBattle records stats](https://docs.google.com/spreadsheets/d/18pO9ESpPTP7w6ERWNPHNSI03CQaSS5S_42nHSo_imbw/edit#gid=0) by @Beowolve currently shows us that the vast majority of top solutions are currently made with Unicode. The latest four Battles (including the ongoing one) are especially remarkable in this regard. Even the infamous (and [apparently ineradicable](https://discord.com/channels/909018468366549003/917706886600618034/1054410262511366266) 😂) "SVG-in-CSS" method is way behind.

To be honest, I have mixed feelings about this.

On the one hand, CSSBattle is about pushing the limits as far as possible, and recreating the shapes with the help of suitable Unicode characters is a good way of doing it. But on the other hand, "CSS code golfing" and "Unicode art golfing" feel like two significantly different games, at least to me. They require different mindset — and even different toolset: Unicode golfing is impossible without the [backend rendering previewer](https://tc70f3.csb.app/). While the "building blocks" of the Unicode art are pretty much predefined — although finding the best fit and best placement is a fairly creative process too — in the "traditional" CSS, small change of code often leads to dramatic changes of the shape, sometime unexpectable and even paradoxical, and such experimentation is an important part of fun when _playing_ with CSS (at least for me, again).

When one method seems clearly superior in the score terms, it can be somewhat demotivating for exploring other ways. When there is virtually no chances, e.g., to beat the Unicode's 88 chars in [#140](https://cssbattle.dev/play/140) with any "traditional" CSS method I could think of, playing with these methods won't pay off much, all findings will remain nearly invisible (apart from sharing them as "fun and interesting solution" in the community).

Can we please reconsider the idea of filtering the solutions by categories like "Best non-Unicode solutions" and "Best non-SVG solutions", in addition to existing "Best solutions of all times"? I guess it would be especially useful for the Open targets: instead of the single confusingly-looking solution currently, players would be able to compare different approaches. Or there will be better suggestions?

**Comment by joewinbpaul** [comment]
Or even better, a battle where we can only use unicode! <p> ok that's probably a bad idea... 😅 😂
